app/app.go

```diff
@@ -526,7 +505,6 @@ func NewStargazeApp(
 	govRouter := legacygovtypes.NewRouter()
 	govRouter.AddRoute(govtypes.RouterKey, legacygovtypes.ProposalHandler).
 		AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(app.ParamsKeeper)).
-		// AddRoute(distrtypes.RouterKey, distr.NewCommunityPoolSpendProposalHandler(app.DistrKeeper)). // COME HERE
```

Comment:
why is this commented out? Is there a different way to do these spend props ?

Reply:
Yea we can do community spend props via v1gov with the following propsal.json so explicit proposal type isn't needed.
```json
{
    "messages": [
     {
      "@type": "/cosmos.distribution.v1beta1.MsgCommunityPoolSpend",
      "authority": "stars13am065qmk680w86wya4u9refhnssqwcv50d96f",
      "recipient": "stars1u3kqrp52yfyaa8w34j43j8f4v9kr9g2a0zulu8",
      "amount": [{
        "denom": "stake",
        "amount": "20"
      }]
     }
    ],
    "metadata": "metadata",
    "deposit": "1000stake",
    "title": "Community spend",
    "summary": "Commuinty spend"
}
```